src_py/sprite.py

```diff
@@ -1332,6 +1333,27 @@ def set_timing_treshold(self, time_ms):
 
         Raises TypeError if time_ms is not int or float.
 
+        """
+        warn('This function will be removed, use set_timing_threshold function instead')
+        if isinstance(time_ms, (int, float)):
+            self._time_threshold = time_ms
+        else:
+            raise TypeError(
```

Comment:
this part can be replaced with `self.set_time_threshold()` to not have code repetition :)